nimbus-features/homescreenFeature.yaml

```diff
@@ -51,9 +51,6 @@ features:
             "status": true,
             "max-number-of-tiles": 2
           },
-          "wallpaper-feature": {
-            "status": true
-          },
```

Comment:
is there a reason this is getting removed? I don't even know what it exactly it controls, but I'm curious if it needs to be removed in a bump PR?

Reply:
So if you look through the HomescreenFeature's property definitions, it doesn't include the `wallpaper-feature` property. The resolution of [this bug](https://mozilla-hub.atlassian.net/browse/EXP-3503) makes it so invalid property keys will throw FML validation errors (of which this is one).

Reply:
I'm also not certain if it should be removed in a bump PR

Reply:
Yeah the validation errors were blocking upgrading AS in https://github.com/mozilla-mobile/firefox-ios/pull/14815 so they needed to be fixed before landing the bump so I put them in the same PR

If it's preferred to separate the deletion out into a separate small one we can do that too (At the very least the bitrise here will tells us that the removal worked in fixing the build)

Reply:
The removal is bound to the bump so I think it's fine like that. We've done that in the past as well when the change needed is small

Reply:
Awesome. Just wanted some context.
LGTM